src/Fhir.Anonymizer.Core/Models/AnonymizationStatus.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace Fhir.Anonymizer.Core.Models
+{
+    public class AnonymizationStatus
+    {
+        public bool IsRedacted { get; set; }
+
+        public bool IsAbstracted { get; set; }
+
+        public bool IsPerturbed { get; set; }
+
+        public void UpdateIsRedacted(string originalValue, string currentValue)
+        {
+            if (!string.Equals(originalValue, currentValue, StringComparison.InvariantCulture))
+            {
+                IsRedacted = true;
+            }
+        }
+
+        public void UpdateIsAbstracted(string originalValue, string currentValue)
+        {
+            if (!string.Equals(originalValue, currentValue, StringComparison.InvariantCulture))
+            {
+                IsAbstracted = true;
+            }
+        }
+
+        public void UpdateIsPerturbed(string originalValue, string currentValue)
+        {
+            if (!string.Equals(originalValue, currentValue, StringComparison.InvariantCulture))
```

Comment:
How about dateshift delta equals to zero?

Reply:
I think we should add security labels in this case as well, as the date is shifted to range [-50, 50] and zero is one of them.